### PR TITLE
Added LSTM

### PR DESCRIPTION
**Pull request by prajwal-144**
fixes #issue_number (if any, otherwise delete this line)

# Description

> :memo: Summary of the change. 
>  Develop a model to classify Commodity Sentiments into predefined categories. 

## Type of change

Note: Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation update
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] New layer/algorithm
- [ ] Other (Specify)

### Implementation details
(only for `New layer/algorithm`)
>numpy == 1.22.4
>pandas == 1.4.2
>seaborn == 0.11.2
>matplotlib == 3.5.1
>sklearn==0.20
>tensorflow==2.2.0
>nltk==3.2.5

- Average time required to train for 1 epoch : 4 seconds running on GPU.

## Tests

>  Obtained an accuracy of 94% on the training set and 90% on the test set.

## Checklist

- [x] I have performed a self code_review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by HemanthSai7 — CHANGES_REQUESTED** [review]
Hey @prajwal-144 👋. Thank you for the PR. Your notebook looks gr8 with good visualizations. 
Few points to be noted:
- You have used TensorFlow's LSTM layer to train the model
- The repository's main aim is not to build a good model but to implement the algorithm from scratch without using any inbuilt functions.
- Please move all your models and images into the Model directory and the Jupyter notebook to the LSTM directory.
Thanks.

**Comment by prajwal-144** [comment]
Okay @HemanthSai7, so should I redo it again?
Like implement the algorithm from scratch?

**Comment by HemanthSai7** [comment]
Yes!

**Comment by mayureshagashe2105** [comment]
@prajwal-144 , Closing due to inactivity. If you wish to continue to work on this, feel free to re-open this PR.
